src/Text/Pandoc/Readers/MediaWiki.hs

```diff
@@ -1,6 +1,4 @@
 {-# LANGUAGE OverloadedStrings #-}
-{-# LANGUAGE RelaxedPolyRec    #-}
```

Comment:
Can you split these deletions of RelaxedPolyRec into a separate PR, since they aren't related?